lab_1_keywords_tfidf/main.py

```diff
@@ -1,72 +1,62 @@
-"""
-Lab 1
-Extract keywords based on frequency related metrics
-"""
 from typing import Optional, Union
 
-
-def clean_and_tokenize(text: str) -> Optional[list[str]]:
-    """
-    Removes punctuation, casts to lowercase, splits into tokens
-
-    Parameters:
-    text (str): Original text
-
-    Returns:
-    list[str]: A sequence of lowercase tokens with no punctuation
-
-    In case of corrupt input arguments, None is returned
-    """
-    pass
-
-
-def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list[str]]:
-    """
-    Excludes stop words from the token sequence
-
-    Parameters:
-    tokens (List[str]): Original token sequence
-    stop_words (List[str]: Tokens to exclude
-
-    Returns:
-    List[str]: Token sequence that does not include stop words
-
-    In case of corrupt input arguments, None is returned
-    """
-    pass
-
-
-def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
-    """
-    Composes a frequency dictionary from the token sequence
-
-    Parameters:
-    tokens (List[str]): Token sequence to count frequencies for
-
-    Returns:
-    Dict: {token: number of occurrences in the token sequence} dictionary
-
-    In case of corrupt input arguments, None is returned
-    """
-    pass
-
-
-def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[list[str]]:
-    """
-    Extracts a certain number of most frequent tokens
-
-    Parameters:
-    frequencies (Dict): A dictionary with tokens and
-    its corresponding frequency values
-    top (int): Number of token to extract
-
-    Returns:
-    List[str]: Sequence of specified length
-    consisting of tokens with the largest frequency
-
-    In case of corrupt input arguments, None is returned
-    """
-    pass
+with open("Дюймовочка.txt", "r", encoding="utf-8") as f1:
+    f2 = open("stop_words.txt", "r", encoding="utf-8")
+    text = f1.read()
+    stopwords = f2.read()
+    stop_words = stopwords.split()
+    tokens = []
+    tokens_clean = []
+    frequencies = {}
+    top = 6
+
+def clean_and_tokenize(text: str):
```

Comment:
You mustn't change arguments and types of arguments or anything has already been doing for you. Revert this changes and place implementation of each function after docstring